users/static/users/center_style.css

```diff
@@ -1,13 +1,101 @@
-table {
-  margin-left:auto;
-  margin-right:auto;
+body{
```

Comment:
Please add space between name and style declaration

Reply:
Done